modules/gdscript/gdscript_parser.cpp

```diff
@@ -3117,6 +3131,14 @@ void GDScriptParser::_parse_block(BlockNode *p_block, bool p_static) {
 							}
 						}
 
+						if (op->arguments.size() == 4 && op->arguments[3]->type == Node::TYPE_CONSTANT) {
+							ConstantNode *c = static_cast<ConstantNode *>(op->arguments[3]);
+							if (c->value.operator double() == 0) {
+								_set_error("Step argument must not be 0.");
```

Comment:
Here's a clearer message which can be useful for long lines:

```suggestion
								_set_error("The range() step argument must not be equal to 0.");
```